packages/onboard-react/src/index.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, waitFor } from "@testing-library/react";
+import { useReducer } from "react";
+import userEvent from "@testing-library/user-event";
+import WalletDialog from "./components/WalletDialog/WalletDialog";
+
+const Component = () => {
+  const [isOpen, toggle] = useReducer((p) => !p, false);
+  return (
+    <>
+      <button onClick={toggle}>Engage</button>
+      <WalletDialog isOpen={isOpen} />
+    </>
+  );
+};
+
+describe("WalletDialog", () => {
+  it("should start with hidden modal", () => {
+    const { getByText } = render(<WalletDialog isOpen={false} />);
```

Comment:
I think you should render `Component` instead of `WalletDialog` directly and try to find "Connect Wallet" text as well. I don't see any "test" in `WalletDialog`.